.github/workflows/express-build.yml

```diff
@@ -0,0 +1,110 @@
+# GitHub Action Workflow for building on Linux and Windows
+
+name: ExpressJS Express Build
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  
+  build:
+    name: Express Build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ["0.10", "0.12", "1.8", "2.5", "3.3", "4.9", "5.12", "6.17", "7.10", "8.17", "9.11", "10.19", "11.15", "12.16"]
+        os: [ubuntu-latest, windows-latest]
+        experimental: [false]
+        node-mirror: ["https://nodejs.org/dist/"]
+        include:
+          - node-version: 13
+            os: ubuntu-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+          - node-version: 13
+            os: windows-latest
+            node-mirror: "https://nodejs.org/download/nightly"
+            experimental: true
+            
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up node using nvm
+      uses: dcodeIO/setup-node-nvm@v4.0.0
+      with:
+        node-version: ${{ matrix.node-version }}
+        node-mirror: ${{ matrix.node-mirror }}
+      
+    # Configure npm
+    # Skip updating shrinkwrap / lock
+    - run: npm config set shrinkwrap false
+      
+    # Remove all non-test dependencies
+    # Remove example dependencies
+    - run: npm rm --silent --save-dev connect-redis
+    
+    - name: Get Major Version Number of NodeJs Using Powershell
+      run: |
```

Comment:
I would be strongly in favor of preferencing `*nix`'isms over relying on powershell here.  It is much more broadly known (I know @dougwilson's experience will disagree here) but [data says otherwise](https://insights.stackoverflow.com/survey/2019#technology-_-developers-primary-operating-systems).

That said, it seems like we could remove all of this if we just restructured the test matrix.  This removes the need to have the conversation of windows vs *nix (for this part at least).

Reply:
I wouldn't disagree, lol. I was also confused why this one part was powershell and not a unix one line the rest.

Reply:
Ok! I thought maybe because you are a windows user and in that session issue you seemed to state otherwise.  No worries :)

Reply:
So that issue was about what to document for our users using the modules, and that we should document both the linux and windows and not simply just one of them.

Reply:
Yep! I agree there is a difference.  So my comment stands, can we get this in a cross-compatible form which does not require powershell?

Reply:
So I answered in the comment below why I used PowerShell Core (sorry got out of order).  The main reason for using PS Core is its cross platform nature allows me to have a step that has scripting in it that can be used on all platforms.

Yes, if we restructured the matrix to only use major integer numbers (4, 5, 6, etc...) then this method isn't needed. It is strictly here to get the major Node number, so I can use that later in some of the conditional statements, to determine what to do.

Reply:
So I "may" have a solution for this, which allows use to stick with *nix'isms, and still run cross platform.  I'll add it with the other changes I'm working on, and see if it works.

Reply:
I've made this change.